virttest/migration_template.py

```diff
@@ -918,12 +919,31 @@ def cleanup(self):
         """
         Cleanup env
         """
-        logging.debug("Start to clean up env")
-        # Shutdown vms
+        logging.info("Start to clean up env")
+
+        # Remove vms on src host(if vm is migrated back)\
+        # or dest host(if vm is not migrated back)
+        logging.info("Remove vms on src or dest host")
         for vm in self.vms:
-            vm.destroy()
+            try:
+                vm.remove()
+            except:
+                pass
+            continue
+
+        # Need to undefine vms on src host(if vm is not migrated back)
+        logging.info("Undefine vms on src host")
+        undef_opts = "--managed-save --snapshots-metadata"
+        if libvirt_version.version_compare(7, 5, 0):
+            undef_opts += " --checkpoints-metadata"
+        for backup in self.vm_xml_backup:
+            try:
+                backup.undefine(options=undef_opts)
+            except:
+                pass
```

Comment:
Could we print any log message for the user to know what happens?

Reply:
OK, I will update